src/schrodinger/rdkit_extensions/helm/monomer_coordgen.cpp

```diff
@@ -760,6 +764,121 @@ static BOND_IDX_VEC get_bonds_between_polymers(const RDKit::ROMol& from,
     }
     return bonds;
 }
+// Assign unit IDs to rings
+static void assign_ring_unit_ids(const RDKit::ROMol& mol,
+                                 std::unordered_map<int, std::string>& unit_ids)
```

Comment:
It would be nice to have documentation for the parameters here (and the next two functions) to explain the contents of `unit_ids`